docs/sources/manage-users/_index.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Manage users"
+type = "docs"
+[menu.docs]
+name = "Manage users"
+identifier = "manage-users"
+weight = 3
++++
+
+# Manage users
+
```

Comment:
We need an intro paragraph here. Something like,

Managing Grafana also means managing user accounts. You can assign permissions to individual users or organize users into teams and assign permissions on a per-team basis. This page describes the entities you will be dealing with.

ARGH, that sucks, but maybe gives you a starting point